pkg/expr/classic/classic_test.go

```diff
@@ -169,6 +169,31 @@ func TestConditionsCmdExecute(t *testing.T) {
 				return v
 			},
 		},
+		{
+			name: "single query and single condition - empty series and not empty series",
+			vars: mathexp.Vars{
+				"A": mathexp.Results{
+					Values: []mathexp.Value{
+						valBasedSeries(),
+						valBasedSeries(ptr.Float64(3)),
+					},
+				},
+			},
+			conditionsCmd: &ConditionsCmd{
+				Conditions: []condition{
+					{
+						QueryRefID: "A",
+						Reducer:    classicReducer("avg"),
+						Operator:   "and",
+						Evaluator:  &thresholdEvaluator{Type: "gt", Threshold: .5},
+					},
+				}},
+			resultNumber: func() mathexp.Number {
```

Comment:
what's the expected output from SSE if all series are null? it seems like it's currently this:

![image](https://user-images.githubusercontent.com/360020/155585468-346ff101-2745-435e-9367-2686ed645de4.png)

should it be an error?

Reply:
Yes it should be Null, because the evaluator (IIRC) uses a nil number to be no data